frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'normal' | 'stretch'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
+  parts: ['outerbox', 'innerbox', 'option'],
+  baseStyle: ({ isActive }) => ({
+    outerBox: {
+      padding: '4px',
```

Comment:
i think we aim to use rem because it's based on element size and hence, more responsive (read more [here](https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/)). for reference, 16px = 1rem.

Reply:
thanks for highlighting - changed to rem